docs/source/topics/known-issues.rst

```diff
@@ -0,0 +1,92 @@
+============================
+Known issues and limitations
+============================
+
+This document summarizes some known issues and limitations of this library.
+
+Like any database, MongoDB has some particularities. Also keep in mind that
+because MongoDB is a NoSQL database, it's impossible to implement SQL-specific
+functionality.
```

Comment:
```suggestion
functionality.
If you notice an issue not listed or would like a feature that has yet to be implemented, visit the Issues & Help section of `docs/source/internals.rst` for instructions on how to file a ticket.
```

Reply:
The above needs verbal workshopping but directing folks, but it's a good idea to still direct folks to where to make requests, especially if they are already looking at the limitations.